bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -187,16 +191,25 @@ for ifo in trigs.ifos:
     data['%s/time' % ifo] = []
     data['%s/trigger_id' % ifo] = []
 
-total_factors = [1]
+factors = [1]
 threshes = [numpy.inf]
-for decstr in args.loudest_keep_values:
+loudest_keep_vals = args.loudest_keep_values.strip('[]').split(',')
+
+for decstr in loudest_keep_vals:
     thresh, factor = decstr.split(':')
     if int(factor) == 1:
```

Comment:
it seems we want some check for non-integer factors .. eg if someone used '1.5' as a decimation factor 
I guess at first order we don't want to allow non-integer, so I would say eg `if int(factor) != float(factor): raise RuntimeError("Sorry, non-integer decimation not supported")

Reply:
Could be that some part of the code does already fail for factor strings like '1.5' - not to mention '-3' ... - but if so it would be nice to see that commented explicitly. Ideally we'd want to explicitly limit the factors to be positive integers